packages/x-scheduler/src/material/timeline/content/TimelineContent.types.ts

```diff
@@ -0,0 +1,5 @@
+import { TimelineView } from '../Timeline.types';
+
+export interface TimelineContentProps extends React.HTMLAttributes<HTMLDivElement> {
+  view?: TimelineView;
```

Comment:
We could move that in the `TimelineState` just like we are doing for the Event Calendar